demo/b2share_demo/data/communities/euon.json

```diff
@@ -41,6 +41,7 @@
                             "type": "array",
                             "items": {
                                 "enum": [
+									"english",
```

Comment:
"english" was a value specified by one of the communities I advice to retain it in the schema this is their choice.